prospero-cli/src/test/java/org/wildfly/prospero/cli/commands/InstallCommandTest.java

```diff
@@ -220,30 +224,33 @@ public void fplAndDefinitionAreNotAllowedTogether() throws Exception {
 
     @Test
     public void provisionConfigAndChannelSet() throws IOException {
-        final File provisionConfigFile = temporaryFolder.newFile();
+        final File channelsFile = temporaryFolder.newFile();
 
         int exitCode = commandLine.execute(CliConstants.Commands.INSTALL, CliConstants.DIR, "test",
-                CliConstants.PROVISION_CONFIG, provisionConfigFile.getAbsolutePath(),
+                CliConstants.CHANNELS, channelsFile.getAbsolutePath(),
                 CliConstants.CHANNEL_MANIFEST, "g:a:v",
                 CliConstants.FPL, "test");
 
         assertEquals(ReturnCodes.INVALID_ARGUMENTS, exitCode);
         assertTrue(getErrorOutput().contains(CliMessages.MESSAGES
-                .exclusiveOptions(CliConstants.PROVISION_CONFIG, CliConstants.CHANNEL_MANIFEST).getMessage()));
+                .exclusiveOptions(CliConstants.CHANNELS, CliConstants.CHANNEL_MANIFEST).getMessage()));
     }
 
     @Test
-    public void provisionConfigAndRemoteRepoSet() throws IOException {
-        final File provisionConfigFile = temporaryFolder.newFile();
+    public void provisionConfigAndRemoteRepoSet() throws Exception {
+        Path channelsFile = temporaryFolder.newFile().toPath();
+        MetadataTestUtils.prepareChannel(channelsFile, List.of(getClass().getClassLoader().getResource("manifest.yaml")));
```

Comment:
maybe instead of adding the test manifest, just mock the manifest file here? just empty temporary file should do I think

Reply:
I can give a non-existing URL as a manifest reference, but the channel file need to exists.